app/scripts/modules/amazon/src/domain/IAmazonFunction.ts

```diff
@@ -0,0 +1,29 @@
+import { IFunction } from '@spinnaker/core';
+
+export interface IAmazonFunction extends IFunction {
+  credentials?: string;
+  role?: string;
+  runtime: string;
+  s3bucket: string;
+  s3key: string;
+  handler: string;
+  functionName: string;
+  publish: boolean;
+  description: string;
+  tags: [{}];
```

Comment:
Is it possible to use a more explicit type or generics for the tags?

Reply:
It looks like `{}` is also equiavalent to `{ [key: string]: any }`, so `{}` may be the best option unless there is a constant, explicit type when there is a tag present